### PR TITLE
Add surface switcher

### PR DESCRIPTION
**Pull request by al1img**
This pull request adds aglsurfaceswitcher application. This application is WA for CES2018 to display android surface over AGL HMI.

### CODE REVIEW

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+SRC_URI_append = " git://github.com/xen-troops/agl_surface_switcher.git;protocol=https;branch=master"
+
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a23a74b3f4caf9616230789d94217acb"
+
+S = "${WORKDIR}/git"
+
+EXTRA_OECMAKE_append = " -DWITH_WGT=ON -DCMAKE_BUILD_TYPE=Release"
+
+DEPENDS = "libxenbe wayland-ivi-extension dbus-cpp git-native dashboard"
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb]: dashbord is not needed

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+SRCREV = "${AUTOREV}"
+DESCRIPTION = "AGL surface switcher"
+SECTION = "extras"
+LICENSE = "GPLv2"
+PR = "r0"
+
+PN = "aglsurfaceswitcher"
+PV = "0.1"
+
+SRC_URI_append = " git://github.com/xen-troops/agl_surface_switcher.git;protocol=https;branch=master"
```
Comment (lorc) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb]: This is first entry, so you don't need append and leading space.

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a23a74b3f4caf9616230789d94217acb"
+
+S = "${WORKDIR}/git"
+
+EXTRA_OECMAKE_append = " -DWITH_WGT=ON -DCMAKE_BUILD_TYPE=Release"
```
Comment (lorc) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb]: Same there.

**Comment by al1img** [comment]
Comments addressed

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+SRCREV = "${AUTOREV}"
+DESCRIPTION = "AGL surface switcher"
+SECTION = "extras"
+LICENSE = "GPLv2"
+PR = "r0"
+
+PN = "aglsurfaceswitcher"
+PV = "0.1"
+
+SRC_URI_append = "git://github.com/xen-troops/agl_surface_switcher.git;protocol=https;branch=master"
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb]: _append

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a23a74b3f4caf9616230789d94217acb"
+
+S = "${WORKDIR}/git"
+
+EXTRA_OECMAKE_append = "-DWITH_WGT=ON -DCMAKE_BUILD_TYPE=Release"
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/aglsurfaceswitcher/aglsurfaceswitcher_git.bb]: _append

**Comment by andr2000** [comment]
LGTM

**Pull request merged**